packages/edit-post/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
-## V.V.V (Unreleased)
+## 3.3.0 (Unreleased)
```

Comment:
Per this week's Core JS chat, we should leave this as unset (or change to "Unreleased" or "Master").